golang/cosmos/app/sim_test.go

```diff
@@ -78,11 +81,19 @@ func TestFullAppSimulation(t *testing.T) {
 		require.NoError(t, os.RemoveAll(dir))
 	}()
 
+	// Use temporary directory to prevent CosmWasm lock conflicts when running tests in parallel
+	homeDir := fmt.Sprintf("/tmp/agoric-test-%d-%d", 1, time.Now().UnixNano())
+
 	appOptions := make(simtestutil.AppOptionsMap, 0)
-	appOptions[flags.FlagHome] = DefaultNodeHome
+	appOptions[flags.FlagHome] = homeDir
```

Comment:
`simtestutil.SetupSimulation` above, returns a temporary scratch directory (`dir`), which we can use to initialize appOptions.

```suggestion
	appOptions := simtestutil.NewAppOptionsWithFlagHome(dir)
```